crates/swc_ecma_minifier/tests/terser/compress/object/property_with_unprintable/output.js

```diff
@@ -1,23 +1,23 @@
 var foo1 = {
-    "\x00\x01": "foo",
-    get "\x00\x01" () {
+    "\0": "foo",
+    get "\0" () {
         return "bar";
     },
-    set "\x00\x01" (foo){
+    set "\0" (foo){
         save(foo);
     },
-    *"\x00\x01" () {
+    *"\0" () {
```

Comment:
This seems problematic

Reply:
Sam here in original code we have `SOH` character, so we just print it as is